lib/decidim/time_tracker/component.rb

```diff
@@ -66,9 +66,32 @@
     # resource.searchable = true
   end
 
-  # component.register_stat :some_stat do |context, start_at, end_at|
-  #   # Register some stat number to the application
-  # end
+  component.register_stat :activities_count, primary: true, priority: Decidim::StatsRegistry::MEDIUM_PRIORITY do |components, start_at, end_at|
+    time_tracker = Decidim::TimeTracker::TimeTracker.find_by(component: components)
+    activities = time_tracker.activities
```

Comment:
```suggestion
    activities = time_tracker.activities.active
```
I'd would use active only no?